extensions/objects/templates/fraction-editor.directive.html

```diff
@@ -1,5 +1,11 @@
-<input type="text" ng-model="$ctrl.localValue.label" class="form-control">
+<input type="text" ng-model="$ctrl.localValue.label" class="form-control" ui-validate="'$ctrl.isValid($value)'">
 
-<div class="oppia-form-error">
+<div class="oppia-form-error fraction-error-text">
```

Comment:
Prefix the name with `oppia-` as suggested [here ](https://github.com/oppia/oppia/wiki/Coding-style-guide#css)in the wiki page.

Reply:
Done!